app/templates/components/question-single.hbs

```diff
@@ -14,6 +17,9 @@
 {{/if}}
 
 <h3 class="moduleHeader">{{question.title}}</h3>
+<div class="questionDescription">
```

Comment:
dasherize the class, like `question-description`. Also update its CSS definition.